airbyte-integrations/connector-templates/generator/plopfile.js

```diff
@@ -25,18 +25,42 @@ module.exports = function (plop) {
   const pythonSourceInputRoot = '../source-python';
   const singerSourceInputRoot = '../source-singer';
   const genericSourceInputRoot = '../source-generic';
+  const pythonCdkSourceInputRoot = '../source-python-cdk';
 
   const outputDir = '../../connectors';
   const pythonSourceOutputRoot = `${outputDir}/source-{{dashCase name}}`;
   const singerSourceOutputRoot = `${outputDir}/source-{{dashCase name}}-singer`;
   const genericSourceOutputRoot = `${outputDir}/source-{{dashCase name}}`;
+  const pythonCdkSourceOutputRoot = `${outputDir}/source-{{dashCase name}}-cdk`;
 
   plop.setActionType('emitSuccess', function(answers, config, plopApi){
       console.log(getSuccessMessage(answers.name, plopApi.renderString(config.outputPath, answers), config.message));
   });
 
+  plop.setGenerator('source-python-cdk', {
+    description: 'Generate a Python CDK Airbyte source.',
+    prompts: [{type: 'input', name: 'name', message: 'Source name, without the "source-" prefix e.g: "google-analytics"'}],
+    actions: [
+      {
+        abortOnFail: true,
+        type:'addMany',
+        destination: pythonCdkSourceOutputRoot,
+        base: pythonCdkSourceInputRoot,
+        templateFiles: `${pythonCdkSourceInputRoot}/**/**`,
+      },
+      // plop doesn't add dotfiles by default so we manually add them
+      {
+        type:'add',
+        abortOnFail: true,
+        templateFile: `${pythonCdkSourceInputRoot}/.dockerignore.hbs`,
```

Comment:
I left the gitignore file out as it was ignoring a file that seemed very specific to the generic python generator. lmk if this should actually be included.